src/pages/index.js

```diff
@@ -138,3 +246,33 @@ const editCardFormValidator = new FormValidator(config, editModalForm);
 
 addCardFormValidator.enableValidation();
 editCardFormValidator.enableValidation();
+
```

Comment:
* The `avatar` form is not validated https://snipboard.io/3VW06a.jpg . Please, create a validator for it